po/de_DE.po

```diff
@@ -0,0 +1,706 @@
+msgstr "Berechne Konflikte…"
+
+#: src/install.rs:1017
+msgid "Calculating inner conflicts..."
+msgstr "Berechne interne Konflikte…"
+
+#: src/install.rs:1029
+msgid "Inner conflicts found:"
+msgstr "Interne Konflikte gefunden:"
+
+#: src/install.rs:1051
+msgid "Conflicts found:"
+msgstr "Konflikte gefunden:"
+
+#: src/install.rs:1074
+msgid "Conflicting packages will have to be confirmed manually"
+msgstr "In Konflikt stehende Pakete müssen manuell bestätigt werden"
+
+#: src/install.rs:1078
+msgid "can not install conflicting packages with --noconfirm"
+msgstr "In Konflikt stehende Pakete können nicht mit --noconfirm installiert werden"
+
+#: src/install.rs:1117
+msgid "Repo"
+msgstr "Repo"
+
+#: src/install.rs:1124
+msgid "Repo Make"
+msgstr "Repo Make"
+
+#: src/install.rs:1138
+msgid "Aur Make"
+msgstr "AUR Make"
+
+#: src/install.rs:1240
+msgid "fetching devel info..."
+msgstr "Frage devel info ab…"
+
+#: src/install.rs:1320
+msgid "Signing packages..."
+msgstr "Signiere Pakete…"
+
+#: src/install.rs:1398 src/install.rs:1403
+msgid "failed to download sources for '{}'"
+msgstr "Quelldateien für ‚{}‘ konnten nicht heruntergeladen werden"
+
+#: src/install.rs:1408 src/install.rs:1447 src/install.rs:1455
+msgid "failed to build '{}'"
+msgstr "‚{}‘ konnte nicht gebaut werden"
+
+#: src/install.rs:1411
+msgid "{}: parsing pkg list..."
+msgstr "{}: Parse pkg Liste…"
+
+#: src/install.rs:1473
+msgid "adding {} to the install list"
+msgstr "Füge {} zur Installationsliste hinzu"
+
+#: src/install.rs:1532
+msgid "could not find package '{:pkg}' in package list for '{:base}'"
+msgstr "Paket ‚{:pkg}‘ konnte nicht in der Paketliste für ‚{:base}‘ gefunden werden"
+
+#: src/install.rs:1670
+msgid "can't find package name in packagelist: {}"
+msgstr "Paketname kann nicht in Paketliste gefunden werden: {}"
+
+#: src/install.rs:1743
+msgid "There are {} members in group"
+msgstr "Es befinden sich {} Einträge in der Gruppe"
+
+#: src/install.rs:1768
+msgid ""
+"\n"
+"\n"
+"Enter a selection (default=all): "
+msgstr ""
+"\n"
+"\n"
+"Geben Sie eine Auswahl an (default=alle): "
+
+#: src/install.rs:1965
+msgid "{}-{} is up to date -- skipping install"
+msgstr "{}-{} ist aktuell — Überspringe Installation"
+
+#: src/keys.rs:47
+msgid "keys need to be imported:)"
+msgstr "Schlüssel müssen importiert werden:)"
+
+#: src/keys.rs:52
+msgid "     {:key} wanted by: {:base}"
+msgstr "{:key} wird gefordert von: {:base}"
+
+#: src/news.rs:57
+msgid "No Date "
+msgstr "Kein Datum"
+
+#: src/news.rs:60
+msgid "No Title"
+msgstr "Kein Titel"
+
+#: src/news.rs:69
+msgid "no new news"
+msgstr "Keine neuen Neuigkeiten"
+
+#: src/query.rs:105
+msgid " [ignored]"
+msgstr "[ignoriert]"
+
+#: src/repo.rs:164
+msgid "failed to get current exe"
+msgstr "Aktuelle exe konnte nicht gefunden werden"
+
+#: src/repo.rs:195
+msgid "syncing local databases..."
+msgstr "Synchronisiere lokale Datenbanken…"
+
+#: src/repo.rs:201
+msgid "  nothing to do"
+msgstr "Nichts zu tun"
```

Comment:
Because of the spaces in front, the string might be used as continuation of a sentence. In this case:
```suggestion
msgstr " nichts zu tun"
```
would be appropriate.

Reply:
This still needs to be fixed.

Reply:
Done, the double whitespace slipped my attention.